IvanZlobinHomeWork1.cpp

```diff
@@ -0,0 +1,59 @@
+#include <cmath>
+#include <iostream>
+#include <windows.h>
+
+int main() {
+  SetConsoleCP(1251);
+  SetConsoleOutputCP(1251);
+  constexpr int Min{2};
+  constexpr int Max{255};
+  constexpr int v1{4};
+  constexpr int v2{2};
+  float a{}, b{}, c{}, x{}, x1{}, x2{};
+
+  std::cout << "ax^2 + bx + c = 0 \n";
+  std::cout << "Please enter number 'a' in range [2..255]:";
```

Comment:
вместо написания в выводе на экран явно чисел, лучше задавать константы. Т.к. если вы например решили поменять константы `Max` то вам прийдется по всему коду менять и это возможный источник ошибок, т.к. вы можете в некоторых местах забыть поменять. 
```suggestion
  std::cout << "Please enter number 'a' in range [" << Min << ".." << Max << "]:";
```